src/components/helpers/formulas.ts

```diff
@@ -0,0 +1,23 @@
+import { UID } from "../..";
```

Comment:
see with mehdi because it's probably related to https://www.odoo.com/odoo/2328/tasks/4589387, but the thrash icon does not work when you enter a range from another sheet

Reply:
It's not related to range from other sheet, in master it simply doesn't work at all :/.
I'll see with mehdi if he already has a fix